mlflow/server/js/src/components/RunView.js

```diff
@@ -133,11 +133,12 @@ class RunView extends Component {
       if (this.state.showNotesEditor) {
         return <NoteEditorView
             runUuid={this.props.runUuid}
+            type={"run"}
```

Comment:
Looks like this is passed in only for a conditional rendering, I'd recommend using composite CSS selector with higher specificity to achieve the same purpose. like `.ExperimentPage .note-view-text-area { ... }`

Reply:
Unfortunately, this is not actually passed in only for conditional rendering - we pass it in so that we know which ID field to look at when making the API call. Our alternative is to simply check which prop is defined, but I think this is more explicit.